packages/runtime-core/__tests__/hydration.spec.ts

```diff
@@ -1571,5 +1574,30 @@ describe('SSR hydration', () => {
       app.mount(container)
       expect(`Hydration style mismatch`).not.toHaveBeenWarned()
     })
+
+    // #9033
+    test('force patch dynamic props when hydrating', () => {
+      const timestamp = Date.now()
```

Comment:
Is there a reason why this test needs to use `Date.now()`? It may be more 'real', but it also makes the test less reliable. The two calls to `Date.now()` might yield the same number, or they might not. Couldn't the same functionality be tested using fixed values instead?

Reply:
I have no problem with using `Date.now()`, but it makes more sense to use a fixed value.